client/state/stats/lists/utils.js

```diff
@@ -760,7 +760,9 @@ export const normalizers = {
 			if ( item.children ) {
 				newRecord.children = item.children.map( ( child ) => {
 					return {
-						label: child.name,
+						// Remove the parent name from the child name.
+						// If the child name is the same as the parent name, use a slash instead.
+						label: child.name?.replace( item.name, '' ) || '/',
```

Comment:
Could we apply the exact regular expression replacement the same as suggestions for another change?